docs/core/macos-prerequisites.md

```diff
@@ -38,14 +38,16 @@ ln -s /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib /usr/local/lib/
 ln -s /usr/local/opt/openssl/lib/libssl.1.0.0.dylib /usr/local/lib/
 ```
 
-Download and install the .NET Core SDK from [.NET Downloads](https://www.microsoft.com/net/download/core). If you have problems with the installation on macOS, consult the [Known issues & workarounds](https://github.com/dotnet/core/blob/master/cli/known-issues.md) topic.
+Download and install the .NET Core SDK from [.NET Downloads](https://www.microsoft.com/net/download/core). If you have problems with the installation on macOS, consult the [1.0.0 Known Issues](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.0-known-issues.md) and [1.0.1 Known Issues](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.0-known-issues.md) topics.
 
 **.NET Core 2.x**
 
-Download and install the .NET Core SDK from [.NET Downloads](https://www.microsoft.com/net/download/core). If you have problems with the installation on macOS, consult the [Known issues & workarounds](https://github.com/dotnet/core/blob/master/cli/known-issues.md) topic.
+Download and install the .NET Core SDK from [.NET Downloads](https://www.microsoft.com/net/download/core). If you have problems with the installation on macOS, consult the [Known issues](https://github.com/dotnet/core/tree/master/release-notes/2.0) topic for the version you have installed.
 
 ## Visual Studio for Mac
 
+You can use any editor of your choice to develop .NET Core applications using the .NET Core SDK. However, if you want to develop .NET Core applications on a Mac in an integrated development environment, you can use [Visual Studio for Mac](https://www.visualstudio.com/vs/visual-studio-mac/). 
```

Comment:
"You can use any editor" means it is "[their] choice." Can the "of your choice" phrase be dropped?

To avoid the "You can ..."; perhaps, just "Use ..." there. .... or something active like "Develop ..." or "Write code ..." or something like that.

Also, I thought that the conjunctive adverb with prep phrase immediately following it wouldn't get a comma (just like two prep phrases back-to-back opening a sentence; which of course is frowned upon by the style manual anyway). Perhaps, it would be simpler to just drop "However." It doesn't seem necessary. I'd suggest ...

With "However" ...

> Use any editor to develop .NET Core applications using the .NET Core SDK. However if you want to develop .NET Core applications on a Mac in an integrated development environment, use \[Visual Studio for Mac](https://www.visualstudio.com/vs/visual-studio-mac/). 

or without "However" ...

> Use any editor to develop .NET Core applications using the .NET Core SDK. If you want to develop .NET Core applications on a Mac in an integrated development environment, use \[Visual Studio for Mac](https://www.visualstudio.com/vs/visual-studio-mac/).